packages/leva/src/components/Leva/LevaRoot.tsx

```diff
@@ -61,6 +61,10 @@ export type LevaRootProps = {
    * If true, the copy button will be hidden
    */
   hideCopyButton?: boolean
+  /**
+   * Change what will be copied when clicking the global copy button
+   */
+  copy?: (values: unknown) => string
```

Comment:
I wonder whether it might sense to group those options similar to the titleBar options?

Reply:
I thought of that, but `hideCopyButton` works for both Input and Title Bar copy button.
But perhaps I could move `copy` into `titleBar`? What do you think?

Reply:
Any thoughts on this? I was planning on finishing the changes for this PR today.